modules/serverless-functions-all-values-in-configmap-to-env-variables.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-all-values-in-configmap-to-env-variables_{context}"]
+= Setting environment variables from all values defined in a config map
+
+. Open the `func.yaml` file for your function.
+
+. For every config map, for which you want to import all key-value pairs as environment variables, add the following YAML to the `envs` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+envs:
+- value: '{{ configMap:myconfigmap }}'
+----
++
+* Substitute `myconfigmap` with the name of the target config map.
+
+. Save the configuration.
```

Comment:
Should you change this to _Save the file._, as is typical in the OpenShift docs, in each of the files in this PR? Or is this style within Serverless?